docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,82 @@
+
+First, you need to choose the time range for the analysis.
+By default, the analysis uses logs from between four weeks ago and the current date, then continues to add new logs to the analysis as they are ingested. You cannot change the time range for the analysis after the machine learning job has been created.
+
+Once you have selected the time range, click *Create ML job* to create the machine learning job.
+Now you can start detecting anomalies in your logs.
+
+[float]
+=== View log entries
+
+Once the machine learning job has been created, the *Analysis* page shows the log entries chart, an overall anomalies chart, and a collapsible entry for the anomalies in each partition.
+
+The time range over which the logs are analyzed is fixed at the time range you selected when you created the machine learning job.
+But you can use the time filter at the top of the *Analysis* page to restrict the period for which the results are shown.
+
+[float]
+=== Log entries chart
+
+[role="screenshot"]
+image::logs/images/analysis-tab-log-entries.png[Analysis tab log entries]
+
+The log entries chart shows an overall visualization of the log entry rate, partitioned and color-coded according to the value of the {ecs-ref}/ecs-event.html[ECS `event.dataset`] field.
+
+You can hover over a time period to see the log rate for each of the partitions for that period.
+
+You can click a partition name on the right hand side to show or hide the values for that partition, or hover over a partition name to highlight just the values for that partition in the chart.
+
+[float]
+=== Overall anomalies chart
+
+[role="screenshot"]
+image::logs/images/analysis-tab-anomalies.png[Analysis tab anomalies]
+
+The overall anomalies chart shows the areas where anomalies were detected in the overall log entry rate across all log partitions. The underlying rate values are shown in grey, and the anomalous regions are color-coded and superimposed on top.
+
+Where a time period is flagged as anomalous, it means that the machine learning algorithms detected something unusual about the log rate in that time period. This may be because the log rate was significantly higher than usual, or significantly lower than usual, or some other anomalous behavior was detected.
+
+The level of anomaly detected in a time period is color-coded from red through orange to yellow and blue, where red indicates a critical anomaly level, and blue is a warning level.
+
+You can hover over an underlying log rate value to see the average log rate for that time period, or hover over an anomalous region to see the partitions that had anomalies in that time period, and their anomaly scores. Anomaly scores range from 0 (no anomalies) to 100 (critical).
+
+You can also click *Analyze in ML* to open the Anomaly Explorer in Machine Learning and {kibana-ref}/xpack-ml.html[analyze the anomalies in more detail].
+
+[float]
+=== Partition anomaly charts
```

Comment:
You'll need to add a link title here.

Reply:
As far as I'm aware, I only need a link title for a section if I plan to link to it. 

The danger in adding link titles unnecessarily is that if you make even a minor change to the content in a section, or remove it entirely, then you have to worry about any content that may have linked to that section (will the link still be appropriate with the changed content? do you need a redirect?). 

I think it's safer only to add a link title to a section if you actually need to link to it. The syntax doesn't require them. And in this particular case, the content is beta, so I'm expecting a lot more changes. I want to avoid unnecessarily worrying about what may have linked to it when I change it.

Reply:
Now that we are using Asciidoctor instead of Asciidoc, we are required to add link titles to each section that have a title. We talked about this during a doc huddle meeting.